docs/modules/user-guide/partials/proc_defining-endpoints.adoc

```diff
@@ -2,7 +2,11 @@
 = Defining endpoints
 
 [role="_abstract"]
-Components of any type can specify the endpoints that the container exposes. These endpoints can be made accessible to the users if the {prod-short} cluster is running using a Kubernetes ingress or an OpenShift route and to the other components within the workspace. You can create an endpoint for your application or database, if your application or database server is listening on a port and you want to be able to directly interact with it yourself or you want other components to interact with it.
+Use any type of component to specify the endpoints that the container exposes. You can access these endpoints if the devfile cluster runs by using a Kubernetes ingress or an OpenShift route. You can also connect these endpoints to the other components within the workspace. Creating an endpoint for your application or database helps you if:
```

Comment:
@jc-berger, why has this been marked as resolved? I still see the wrong use of the `{prod-short}` attribute.

Reply:
Thought you wanted me to return to the original phrasing. I've removed `{prod-short}` all together so the sentence is simply, `Users can access these endpoints if the cluster is running using a Kubernetes ingress...`

Reply:
@jc-berger, returning to "original phrasing" makes no sense because the `{prod-short}` attribute resolves to `devfile` -- therefore, there is no change between your use of `devfile` and the orig. version. The point is that the content talks about a cluster used to run Eclipse Che (the text was taken from Che docs where `{prod-short}` would resolve to `Che`).

So, this requires more than just changing (or omitting) single words. The content needs to be adapted to suit devfile docs (where there might be a mention of an app-specific cluster -- or perhaps there shouldn't be any mention of it at all; this needs to be confirmed with SMEs).

Regardless, we cannot talk about "devfiles cluster" (there's no such thing), nor can we talk about "a cluster" (what cluster, where did it come from what purpose it serves in the doc?).

Reply:
So, I wonder if something very simple, like what we have in our migration docs, would be better for this doc (and maybe all of the authoring docs), a one liner, like `This section describes how to define endpoints and specify their properties.` 

In case we do want to keep more to the original content, I've revised it again. I've removed the reference to clusters and databases (we never talk about databases in any of our other devfile docs). I also shortened it because it was wordy. Basically, it only entails endpoints and ports. 

@maysunfaisal, would you please give us a SME opinion?  Do you think what I have in the latest commit is good or maybe something bare bones like, `This section describes how to define endpoints and specify their properties.` 

Thanks!